### ISSUE TITLE
Pagination: 'Complex' type doesn't announce current page to Screen Readers on page change

### ISSUE THREAD

**Issue by GCHQ-Developer-299** [mi6/ic-ui-kit#1808]
## Summary 
When interacting with a [simple Pagination component](https://mi6.github.io/ic-ui-kit/branches/develop/web-components/?path=/story/web-components-pagination--simple) using a screen reader, the current page is announced to the user whenever a next/prev page button is clicked. This is because in Simple Paginator the page label has `aria-live="polite"` on it, so it announces whenever the label changes.

With the [complex Pagination component](url) this is not the case. Instead users have to navigate back through the page links to see which one is marked as "current item". This leads to an inconsistent experience - please make the current page announce itself to users.

## 💬 Description 
Suggestion: create a text label for complex paginator that's visible only to screenreaders (maybe with `class="sr-only"`) that updates on page change like simple paginator does.

## 💰 Use value 
Parity for screen reader experience
